ddsrouter_test/compose/test_cases/dds/compose.yml

```diff
@@ -99,9 +99,9 @@ services:
       - std_net
     command: install/AdvancedConfigurationExample/examples/cpp/dds/AdvancedConfigurationExample/AdvancedConfigurationExample publisher --interval 100 --samples 80 --domain 4 --topic topic3
 
-  subscriber_t0:
+  subscriber_t9:
```

Comment:
`tX` referes to the topic on which the publisher/subscriber publishes/receives data. My recommendation is to rename all of them to something like `publisher_1_t1` so every container have a unique name given by the publisher/subscriber Id (middle number)